content/app/github/wrappers/file-element-wrapper.js

```diff
@@ -37,6 +39,18 @@ define("content/app/github/wrappers/file-element-wrapper", function(require, exp
 		 */
 		constructor : function (domElement) {
 			FileElementWrapper.$super.call(this, domElement);
+			this._fileName = null;
+			this._init();
+		},
+
+		/**
+		 * File name.
+		 * @type {String}
+		 */
+		fileName : {
+			get : function() {
+				return this._fileName;
+			}
 		},
```

Comment:
We should consult about that. I've put file name as a simple getter, since it will not be changed during page life cycle. However, situation with file-data is different, since uncollapsing will change the 'model'. Because of that, you advised that we use **get** methods which indicates some processing. For me, I prefer simplicity and to always use getters for greater readability. However, current approach also makes sense.